### PR TITLE
Passing database name 1

### LINKED ISSUE #14: Make it support the passing in of a database name
# Purpose

To allow the passing of the MTK_DUMP_* variables into the task in one way or another, rather than assuming that the database is going to be the one in the Lagoon service.  This is useful in some cases.  

Also instead allow the passing of the name of a variable that contains the MTK_DUMP_* value. For example,

MTK_DUMP_HOSTNAME_NAME=DB_HOSTNAME_CENTRAL

... means that MTK_DUMP_HOSTNAME will be set to the value of DB_HOSTNAME_CENTRAL

### PR DESCRIPTION
**Pull request by tnelson-doghouse**
Purpose
-------
-	To allow the passing of the MTK_DUMP_* variables into the task in one
	way or another, rather than assuming that the database is going to be
	the one in the Lagoon service
-	Also instead allow the passing of the name of a variable that contains
	the MTK_DUMP_* value.  For example,

		MTK_DUMP_HOSTNAME_NAME=DB_HOSTNAME_CENTRAL

	... means that MTK_DUMP_HOSTNAME will be set to the value of
	DB_HOSTNAME_CENTRAL

Primary Changes
---------------
-	Added calc_MTK_variable function which sets up one MTK_DUMP_* variable
-	Set all the MTK_DUMP_* variables using the above function (and replaced
	existing code that set them)
-	Changed GraphQL example to allow for the new variables

Other changes
-------------
-	Removed san_db_tarfile line since it's not used
-	A few tweaks based on the output of https://www.shellcheck.net/ (which
	you can install locally)
- Added a little more to the README

closes #14

### CODE REVIEW

**Comment by tnelson-doghouse** [comment]
For the record, this one was discussed with Sean.  Our plan is that:
- The new variables will only be used for Captivate and similar projects that have a multi-tenant setup
- Most of the variables will be hardcoded somewhere; I'm thinking the task GQL would be a good place
- We can have the Dev or Ops team pass in the database name so that we can dump a sanitised database which /isn't/ configured in Lagoon

HTH,

File: createDumpSanitisedDB.gql
Comment (shreddedbacon) [inline createDumpSanitisedDB.gql:1]: Can you create a copy of this file and revert the changes to this. Create a new one with just these changes in it with a different name.

**Review by shreddedbacon — CHANGES_REQUESTED** [review]
I'm fine with this. I'd just like the example graphql command split to its own example file.
